### PR TITLE
feat(roles): Team-roles for Project Creation page

### PR DESCRIPTION
**Pull request by leedongwei**

### CODE REVIEW

**Comment by leedongwei** [comment]
There's some overlap with https://github.com/getsentry/sentry/pull/49740

File: static/app/views/replays/list/replayOnboardingPanel.tsx [hunk 1]
```diff
@@ -178,7 +180,7 @@ export function SetupReplaysCTA({
       <Tooltip
         title={
           <span data-test-id="create-project-tooltip">
-            {t('Only admins, managers, and owners, can create projects.')}
+            {t('You do not have permission to create a project.')}
```
Comment (sentaur-athena) [inline static/app/views/replays/list/replayOnboardingPanel.tsx:183]: 😆 smart

File: static/app/views/settings/organizationProjects/createProjectButton.tsx [hunk 2]
```diff
@@ -2,20 +2,19 @@ import {Button} from 'sentry/components/button';
 import {useProjectCreationAccess} from 'sentry/components/projects/useProjectCreationAccess';
 import {IconAdd} from 'sentry/icons';
 import {t} from 'sentry/locale';
-import {Organization} from 'sentry/types';
+import useOrganization from 'sentry/utils/useOrganization';
+import {useTeams} from 'sentry/utils/useTeams';
 
-type Props = {
-  organization: Organization;
```
Comment (sentaur-athena) [inline static/app/views/settings/organizationProjects/createProjectButton.tsx:8]: Thanks for fixing this 😅

**Review by sentaur-athena — APPROVED** [review]
Frontend changes look good to me. @leedongwei is there anything in the backend that might stop the team admins from project creation?

**Comment by leedongwei** [comment]
@sentaur-athena The API inherits from `ProjectEndpoint` so it'll work for team-admins out of the box.

**Pull request merged**